main/modemmanager/files/modemmanager

```diff
@@ -0,0 +1,9 @@
+# modemmanager service
+
+type            = process
+command         = /usr/bin/modemmanager
+before          = network.target
+depends-on      = dbus
+depends-on      = pre-network.target
+restart         = true
+smooth-recovery = true
```

Comment:
```suggestion
type = process
command = /usr/bin/modemmanager
before = network.target
depends-on = dbus
depends-on = pre-network.target
smooth-recovery = true
```

don't need indents or restart=true anymore (yeah should update the other files eventually..)